### PR TITLE
Update workflow descriptions with escape characters for makrdown support

### PR DESCRIPTION
**Pull request by amouhoub**

### CODE REVIEW

File: ApplicationConnectors/resources/catalog/Run_SSIS_Package.xml [hunk 1]
```diff
@@ -7,9 +7,9 @@
   </variables>
   <description>
     <![CDATA[ A workflow that runs an SSIS package. Before the execution, the user has to provide the following variables:
-SSIS_PACKAGE_PATH (required) is the full path of the SSIS package deployed on the Integration Services server. For example, \SSISDB\Test Package\testSSIS\Package.dtsx.
-SERVER_NAME (required) is the name of the SQL Server instance. The SERVER option is used with the ISSERVER option. Only Windows Authentication can run a package on the SSIS server. Access to the package is through the current Windows user. If the Server option is omitted, the default local instance of SQL Server is used.
-EXECUTION_TOOL_PATH (required) is the path of the dtexec tool used to run SSIS packages either in 32 bit mode or in 64 bit mode. For example, C:\Program Files\Microsoft SQL Server\150\DTS\Binn.
```
Comment (fviale) [inline ApplicationConnectors/resources/catalog/Run_SSIS_Package.xml:12]: actually, it must stay Binn

right @aminelouati ?

Comment (aminelouati) [reply ApplicationConnectors/resources/catalog/Run_SSIS_Package.xml:12]: Yes, I confirm @fviale

**Review by fviale — APPROVED** [review]

**Comment by fviale** [comment]
Ok for me but please leave Binn where it's supposed to be

File: ApplicationConnectors/resources/catalog/Run_SSIS_Package.xml (hunk 1 above)
Comment (amouhoub) [reply ApplicationConnectors/resources/catalog/Run_SSIS_Package.xml:12]: Great finding @fviale how could you spot it ? 😮

**Pull request merged**